### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by MaraJade**

### CODE REVIEW

File: docs/source/info.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.. _info:
+
+Information
+===========
+
+The formsender app is a simple program that is designed to aid in the collection
+of form data. When a form is submitted, the app gathers the data from it, puts
+it in a python dictionary, then sends the dictionary to a specified email.
```
Comment (Kennric) [inline docs/source/info.rst]: Well, it sends a nicely formatted email with the contents of the dictionary, this sounds a bit like it emails a raw python dict. I don't think you need to mention dictionary at all, that's just an implementation detail, the main thing to say is that it takes the data from the form fields and emails it to a configured address. (Addresses? does formsender handle a list of email destinations?)

File: docs/source/usage.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.. _usage:
+
+How To Use Formsender
+=====================
+
+At the bottom of request_handler.py there is a command called run_simple. The
+first variable is the location of the form, the second is the port. Change those
+two variables to match your setup.
+
```
Comment (Kennric) [inline docs/source/usage.rst]: If we're changing variables, they should be in the config file. (fix this in a new code pr)

File: docs/source/usage.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    TOKN = u'1234567890-='
+    CEILING = 10
+
+* ``EMAIL`` is where the form data will be sent.
+* ``TOKN`` is the validating token from the form. This must match a hidden field
+  in your form called 'tokn'.
+* ``CEILING`` is the maximum number of submit requests formsender will accept
+  per second.
+
+To start the application:
```
Comment (Kennric) [inline docs/source/usage.rst]: Well, not really, this should be installed under a web server and run by the wsgi server. This way of running it is just for dev purposes.

File: docs/source/usage.rst (hunk 2 above)
Comment (matthewrsj) [inline docs/source/usage.rst]: @Kennric this is just where the app is listening. This will change when we set it up in a docker environment.

**Comment by Kennric** [comment]
+1

**Pull request merged**